changes/14464-add-kolide-tables

```diff
@@ -0,0 +1,29 @@
+- added tables to the fleetd extension: 
+  - airport_util
+  - app_icons
+  - apple_silicon_security_policy
+  - falconctl_options
+  - falcon_kernel_check
+  - cryptoinfo
+  - cryptsetup_status
+  - parse_plist
+  - parse_json
+  - parse_jsonl
+  - parse_xml
+  - parse_ini
```

Comment:
```suggestion
  - parse_plist
  - parse_json
  - parse_jsonl
  - parse_xml
  - parse_ini
```

Noah: These tables are dependencies for other tables right?

Are we adding any of the tables that depend on them? If not, let's cut these.

Reply:
crowdstrike, cryptoinfotable, cryptsetup, firefox_preferences, and ioreg all have dependencies on this functionality, but they don't require the tables to be imported if that makes sense.

Reply:
> crowdstrike, cryptoinfotable, cryptsetup, firefox_preferences, and ioreg all have dependencies on this functionality, but they don't require the tables to be imported if that makes sense.

Thanks Tim!

If we pull out `parse_plist`, `parse_json`, `parse_jsonl`, `parse_xml`, and `parse_ini` will the above tables still work?

Reply:
yes indeed

Reply:
I think we pull them out then! Less to maintain